includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1605,6 +1605,16 @@ public static function add_admin_notices() {
 	public static function get_error_details_json( $term ) {
 		$json = json_decode( $term->description, true );
 
+		if ( isset( $_GET['post'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			$validation_errors = AMP_Validated_URL_Post_Type::get_invalid_url_validation_errors( (int) $_GET['post'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotValidated
+			foreach ( $validation_errors as $error ) {
+				if ( isset( $error['data']['sources'], $error['term']->term_id ) && $error['term']->term_id === $term->term_id ) {
```

Comment:
I realize here that it's possible that the sources may not be specifically for this term. Specifically, it's possible for an error to occur multiple times on a URL, and each time the source may different. That does complicate things here, however, since the `{{$taxonomy}}_row_actions` filter doesn't pass any index. Perhaps that could be captured by by setting an index to `0` before rendering the table? 

Then with each invocation of `filter_tag_row_actions` the index could be incremented. The index can then be provided from `filter_tag_row_actions` to `get_error_details_json` in addition to the `$term` (which would actually be redundant here), since the error could then be obtained in `get_error_details_json` here via `$validation_errors[ 
$index ]`.

Reply:
@westonruter I think I have it resolved in https://github.com/ampproject/amp-wp/pull/5835/commits/6af38a9484e21461df55c0961844d94828b28cd1.